pom.xml

```diff
@@ -8,7 +8,7 @@
     <parent>
         <groupId>io.confluent</groupId>
         <artifactId>common</artifactId>
-        <version>7.7.0-286</version>
+        <version>7.7.0-351</version>
```

Comment:
@sambhav-jain-16 I don't think this supposed to work this way, rest-utils version is bumped automatically from ce-kafka -> common.

I've also left a comment on the PR in confluent-security-plugins https://github.com/confluentinc/confluent-security-plugins/pull/701/files#r1458741416

Reply:
Thanks @trnguyencflt for helping with this. I agree with the comment you posted in the PR. I'll close this and raise a PR in the other repo with the appropriate fix.